browser/src/view/components/Content/ContentCollection.jsx

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import './ContentCollection.scss';
+import { Skeleton } from '../Skeleton/Skeleton';
+
+export const ContentCollection = () => {
+  return (
+    <table>
+      <colgroup>
+        <col className='accounts'></col>
+        <col className='description'></col>
+        <col className='date'></col>
+        <col className='labels'></col>
+        <col className='status'></col>
```

Comment:
You may want to namespace your classes a bit here.  `date` is very very generic.  One of the benefits of SCSS is you can use the `&` as a parent selector reference.

Say you add a `content-collection` class to the table.  In your colgroups, you can do something like `className="content-collection__accounts"`.  Then when you're in scss you can do this:

```scss
.content-collection {
  &__accounts {
    // whatever rules
  }
}
```

This namespaces your SCSS and is pretty DRY.